src/components/Inventory/index.tsx

```diff
@@ -34,7 +23,8 @@ import { downloadInventoryTemplate } from 'src/api/inventory/inventory';
 import ImportInventoryModal from './ImportInventoryModal';
 import { Button } from '@terraware/web-components';
 import PageHeaderWrapper from 'src/components/common/PageHeaderWrapper';
-import Icon from '../common/icon/Icon';
+import { DropdownItem } from '@terraware/web-components/components/Dropdown';
```

Comment:
as much as possible we should import from '@terraware/web-components' only. can we add a task to export DropdownItem from web-components index.ts?